packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -863,6 +863,8 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
       color: enabled ? decorationColor : (decorationColor ?? disabledColor),
     );
 
+    final Color selectionColor = CupertinoTheme.of(context).primaryColor.withOpacity(0.2);
```

Comment:
Is there anywhere you can document what the selection color will be for Material and Cupertino?